builder/vmware/iso/step_create_vmx.go

```diff
@@ -415,6 +420,9 @@ config.version = "8"
 virtualHW.productCompatibility = "hosted"
 virtualHW.version = "{{ .Version }}"
 
+// Virtual Hardware-Assisted Virtualization
+vhv.enable = {{ .HardwareAssistedVirtualization }}
```

Comment:
Instead of the map you can let the template engine determine what string to write. 
```suggestion
vhv.enable = "{{if .HardwareAssistedVirtualization -}} TRUE {{- else -}} FALSE {{- end -}}"
```